platform-tests/src/test/java/org/junit/platform/console/options/ConsoleUtilsTests.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.console.options;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.charset.Charset;
+
+import org.junit.jupiter.api.Test;
+
+class ConsoleUtilsTests {
+
+	@Test
+	void consolesCharsetIsEither() {
+		var console = System.console();
+		var expected = console != null ? console.charset() : Charset.defaultCharset();
```

Comment:
Isn't `System.console` always `null` here when running tests?

Reply:
...it depends. ;-)